README.md

```diff
@@ -1,24 +1,79 @@
+sonar.projectKey=<replace with the key generated when setting up the project on SonarCloud>
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=windows-msbuild-gh-actions-sq
+#sonar.projectVersion=1.0
+sonar.exclusions=venv/**
+
+# relative paths to source directories. More details and properties are described
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
+sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
+```
+The workflow, usually declared in `.codecatalyst/workflows/build.yaml`, looks like:
+
+```yaml
+Name: SonarCloudAnalysis
+SchemaVersion: "1.0"
+
+# Optional - Set automatic triggers.
+Triggers:
+  - Type: Push
+# Required - Define action configurations.
+Actions:
+  SonarCloudScanAction:
+    # Identifies the action. Do not modify this value.
+    Identifier: sonar/sonarcloud-scan@v1.0.0
+    # Specifies the source and/or artifacts to pass to the action as input.
+    Inputs:
+      # Required
+      Sources:
+        - WorkflowSource # This specifies that the action requires this Workflow as a source
+    Compute:
+      Type: EC2
+    # Defines the action's properties.
+    Configuration:
+      SonarToken: ${Secrets.SONAR_TOKEN}
+```
+
+When your workflow runs, the **SonarCloud Scan** will execute and send analysis results to [SonarCloud](https://sonarcloud.io/).
+
+
+## Configuration parameters
+
+- `SonarToken` – **Required** this is the token used to authenticate access to SonarCloud. You can generate a token on your [Security page in SonarCloud](https://sonarcloud.io/account/security/). You can set the `SONAR_TOKEN` environment variable in the "Secrets" settings page of your repository.
+- `SonarOrganization` – **Optional** The key of the SonarCloud organization to which the project belongs. If not provided, the organization key will be read from the `sonar-project.properties` file.
+- `SonarProjectKey` – **Optional** The key of the project on SonarCloud. If not provided, the project key will be read from the `sonar-project.properties` file.
+- `SonarBranchName` – **Optional** The name of the branch being analyzed.
+- `SonarProjectBaseDir` – **Optional** The base directory of the project to analyze. If not provided, the base directory will be the root of the repository.`
+- `SonarAdditionalArgs` – **Optional** Additional arguments to pass to the SonarScanner CLI. For example, `-Dsonar.verbose=true` to enable verbose logging.
 
 ## Do not use this CodeCatalyst action if you are in the following situations
 
-* Your code is built with Maven: run 'sonar:sonar' during the build
-* Your code is built with Gradle: use the Sonar plugin for Gradle during the build
+* Your code is built with Maven: run 'org.sonarsource.scanner.maven:sonar' during the build
+* Your code is built with Gradle: use the SonarQube plugin for Gradle during the build
```

Comment:
```suggestion
* Your code is built with Gradle: use the SonarScanner for Gradle during the build (more info in the [SonarScanner for Gradle](https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/sonarscanner-for-gradle/) documentation)
```

Reply:
What about removing what is between parenthesis and just having the link on `SonarScanner for Gradle` ?

Reply:
OK. Let's do that.